libcontainer/specconv/spec_linux_test.go

```diff
@@ -154,7 +156,7 @@ func TestSetupSeccomp(t *testing.T) {
 			expectedCloneSyscallArgs := configs.Arg{
 				Index:    0,
 				Op:       7, // SCMP_CMP_MASKED_EQ
-				Value:    2080505856,
+				Value:    2114060288,
```

Comment:
Why not change this to the `unix.*` values you have listed above?

Reply:
@cyphar  There is no meaning in that. I should use unix.* value as you said.